base/db/R/db_merge_into.R

```diff
@@ -11,7 +11,7 @@
 #' db_merge_into(iris[1:12,], "iris", irisdb)
 #' dplyr::tbl(irisdb, "iris") %>% dplyr::count()
 db_merge_into <- function(values, table, con, by = NULL, drop = FALSE, ...) {
-  values_fixed <- match_dbcols(values, table, con, drop = FALSE)
+  values_fixed <- match_dbcols(values, table, con, drop = drop)
```

Comment:
These changes to `db_merge_into` seem like they deserve their own PR. I can't infer from the code whether this is a bug fix, an interface extension, or an ensemble-specific change, and even if it is ensemble-specific it will be helpful to future maintainers of the DB functions if this has its own history that documents:

* Why the change was needed 
* How it changes function behavior
* If fixing a bug, there should be a link to an associated issue number and new tests that cover the bug case
* If changing the existing behavior of `db_merge_into` to accommodate  the ensemble case, how do we demonstrate that `db_merge_into` still behave correctly for non-ensemble cases? Tests would be ideal here, but I realize adding them post facto is often hard to do.

Reply:
Well... This was not specific to ensemble really and it was mainly a bug fix. The two arguments of by and drop are never used in the code. I ran into a problem with the last inner join in this function which without the by tries to join the two tables using all the columns.
You're right though, I stopped using this function because it needs a separate PR with more changes and also it slows down the write.ensemble.configs.